File: leetcode/easy/2490/2490.js
/**
 * @param {string} sentence
 * @return {boolean}
 */
var isCircularSentence = function(sentence) 
{
    var words = sentence.split(" ");
    for (var i = 0; i < words.length-1; i++)
    {
        if (words[i][words[i].length-1] != words[i+1][0])
        {
            return false;
        }
    }
    if (words[words.length-1][words[words.length-1].length-1] != words[0][0])
    {
        return false;
    }
    return true;
};

File: leetcode/easy/2490/2490.go
func isCircularSentence(sentence string) bool {
	str := strings.Split(sentence, " ")
	for i := 0; i < len(str)-1; i++ {
		if str[i+1][0] != str[i][len(str[i])-1] {
			return false
		}
	}
	i := len(str)-1
	if  str[i][len(str[i])-1] != str[0][0] {
		return false
	}
	return true
}